### ISSUE TITLE
ansible.builtin.include has been Removed in Ansible version 2.16

### ISSUE THREAD

**Issue by 0hlov3** [ansible-ThoTeam/nexus3-oss#389]
Hi there,

the ansible.builtin.include has been Removed in the newest version of Ansible.
```
Why: it has too many conflicting behaviours depending on keyword combinations and it was unclear how it should behave in each case. new actions were developed that were specific about each case and related behaviours.
```

The Role should be changed, so that the Role just use one of:
[include_role module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_role_module.html#ansible-collections-ansible-builtin-include-role-module) – Load and execute a role
[include_tasks module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_tasks_module.html#ansible-collections-ansible-builtin-include-tasks-module) – Dynamically include a task list
[include_vars module](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/include_vars_module.html#ansible-collections-ansible-builtin-include-vars-module) – Load variables from files, dynamically within a task

EDIT:

- https://github.com/ansible/ansible/blob/v2.16.0/changelogs/CHANGELOG-v2.16.rst
  -   Removed `include` which has been deprecated in Ansible 2.12. Use `include_tasks` or `import_tasks` instead.

This is just an Information for the Roadmap. :)

**Comment by zeitounator** [comment]
Hi @0hlov3, sorry for the delay and thanks for your interest.

Can you please check which version of the role you are running? I'm a bit confused because this as been fixed ages ago and all tasks in this role are using `import_*` or `include_*` since [v2.2.0](https://github.com/ansible-ThoTeam/nexus3-oss/releases/tag/v2.2.0) released in 2018 (see #81) and have been since converted to full fqcn usage. Unless my search foo has gone really bad, I did not find any `include:`or `ansible.builtin.include:` in the current code (and I'm almost sure `ansible-lint` which runs on every CI build would have told me anyway).

If I'm wrong (which can always happen...), can you please let me know in which task you have found the problem exactly ? Thanks and cheers.

**Comment by zeitounator** [comment]
Without a return, I'm closing this for now. Please comment and tag me directly in case there is still an include you found somewhere.
